pkg/setting/settings_zanzana.go

```diff
@@ -12,20 +12,24 @@ const (
 	ZanzanaModeEmbedded ZanzanaMode = "embedded"
 )
 
-type ZanzanaSettings struct {
-	// Addr is only used when mode is set to client
-	Addr string
-	// Mode can either be embedded or client
+type ZanzanaClientSettings struct {
+	// Mode can either be embedded or client.
 	Mode ZanzanaMode
```

Comment:
ZanzanaMode feels like more general option, but I think we can leave it here for now.

Reply:
Yes I agree, I was thinking of having a general section first but figured this was probably fine